test/readonly_test.rb

```diff
@@ -77,8 +78,10 @@ def assert_readonly_fetch
 
   def assert_readonly_fetch_multi
     cas_multi = Spy.on(backend, :cas_multi).and_call_through
+    write = Spy.on(backend, :write).and_call_through
     yield
     assert(cas_multi.has_been_called?)
+    refute(write.has_been_called?)
```

Comment:
We set the cache to read-only (aka, should_fill_cache is false), so we don't call `write` method.